weave-js/src/components/ErrorBoundary.tsx

```diff
@@ -25,6 +26,12 @@ export class ErrorBoundary extends Component<Props, State> {
       'weave_panel_error_boundary',
       weaveErrorToDDPayload(error)
     );
+
+    Sentry.captureException(error, {
```

Comment:
Do you happen to have a link to sentry docs for this API? I was wondering whether the tag value had to be a string and I couldn't find them.

Reply:
https://docs.sentry.io/platforms/javascript/enriching-events/tags/

"Tags are key/value string pairs"

Let me find the other one rq

Reply:
https://docs.sentry.io/platforms/javascript/enriching-events/context/#passing-context-directly